QCReport/IDseq QC Report.ipynb

```diff
@@ -1141,7 +1390,7 @@
     "    ax3.set_xlabel('log10(Input Mass (ng))')\n",
     "    cd = sns.color_palette(\"Reds\", 11)\n",
     "    col_index = [i if not math.isnan(i) else 0 for i in np.round(rsq_coefs_series.loc[report_csv_df.index]*10)]\n",
-    "    np.log10(report_csv_df['input_mass'] + 1).plot(kind='barh', width=.8, ax=ax3, color=[cd[int(i)] for i in col_index])\n",
+    "    np.log10(rsq_eqn_df['input_mass'] + 1).plot(kind='barh', width=.8, ax=ax3, color=[cd[int(i)] for i in col_index])\n",
```

Comment:
The "input_mass" (in ng) is now saved in the ERCC-related values file (rsq_eqn.csv) as opposed to the report_csv.